### PR TITLE
Diabetes

### PR DESCRIPTION
**Pull request by bottlerocket**

### CODE REVIEW

File: views/conditions/type-2-diabetes/getting-diagnosed.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Getting diagnosed - Type 2 diabetes - - NHS.UK Alpha" %}
+
+{% block localheader %}
+  <div id="local-header">
+
+    <div class="header-local-container">
+        <h1>Type 2 diabetes<</h1>
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/getting-diagnosed.html]: One to many `<` in there

File: views/conditions/type-2-diabetes/check-if-you-have-it.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                  Check if you have it
+              </li>
+              <li class="nav-item">
+                  <a href="getting-diagnosed">Getting diagnosed</a>
+              </li>
+              <li class="nav-item">
+                 <a href="understanding-medication">Understanding medication</a>
+              </li>
+              <li class="nav-item">
+                  <a href="going-for-regular-check-ups">Going for regular check ups/a>
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/check-if-you-have-it.html]: Missing a `<`

File: views/conditions/type-2-diabetes/getting-diagnosed.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  <div class="content">
+
+      <div class="grid-row">
+
+        <div class="grid-2-3 pagination-container">
+
+                <div class="grid-1-2 article-pagination -previous">
+                    <a href="check-if-you-have-it" >
+                    <div class="prefix">Previous page:</div>
+                    <div class="title">Seeking help</div>
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/getting-diagnosed.html]: Shouldn't this be "Check if you have it"?

File: assets/scss/modules/header-local/_header-local.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #local-header {
   color: $white;
-  background-color: $light-blue;
+  background-color: $dark-blue2; /* $light-blue;*/
```
Comment (Demotive) [inline assets/scss/modules/header-local/_header-local.scss:3]: This is nicer 😄

**Comment by Demotive** [comment]
Why is `views/conditions/depression-and-anxiety/getting-therapy.html` being totally deleted?

This was the file I was talking about. I thought i'd forgotten to do it last week.

File: views/conditions/type-2-diabetes/check-if-you-have-it.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Check if you have it - Type 2 diabetes - - NHS.UK Alpha" %}
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/check-if-you-have-it.html]: `Type 2 diabetes - - NHS.UK Alpha` - there's a double dash there?

File: views/conditions/type-2-diabetes/finding-help-and-support.html [hunk 6]
```diff
@@ -0,0 +1,153 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Finding help and support - Type 2 diabetes - - NHS.UK Alpha" %}
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/finding-help-and-support.html]: Same again, double dash...

File: views/conditions/type-2-diabetes/finding-help-and-support.html [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+{% block content %}
+
+<div class="content">
+
+    <div class="grid-row">
+
+      <div class="grid-2-3">
+
+          <h2 class="heading-xlarge" id="finding-help-and-support-for-type-2-diabetes">Finding help and support for type 2 diabetes</h2>
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/finding-help-and-support.html:50]: Do we need `id` attributes for the headers? It's not a big deal either way in this repo.

File: views/conditions/type-2-diabetes/food-and-keeping-active.html [hunk 8]
```diff
@@ -0,0 +1,144 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Food and keeping active - Type 2 diabetes - - NHS.UK Alpha" %}
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/food-and-keeping-active.html]: Double dash

File: views/conditions/type-2-diabetes/getting-diagnosed.html [hunk 9]
```diff
@@ -0,0 +1,144 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Getting diagnosed - Type 2 diabetes - - NHS.UK Alpha" %}
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/getting-diagnosed.html]: Double dash

File: views/conditions/type-2-diabetes/finding-help-and-support.html (hunk 7 above)
Comment (bottlerocket) [inline views/conditions/type-2-diabetes/finding-help-and-support.html:50]: It was a gross copy and paste from the original files. There may have been logic there that I didnt want to arbitrarily over-write.. ;-)

File: views/conditions/type-2-diabetes/going-for-regular-check-ups.html [hunk 10]
```diff
@@ -0,0 +1,129 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Going for regular check ups - Type 2 diabetes - - NHS.UK Alpha" %}
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/going-for-regular-check-ups.html]: Double dash

File: views/conditions/type-2-diabetes/health-problems.html [hunk 11]
```diff
@@ -0,0 +1,177 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Health problems - Type 2 diabetes - - NHS.UK Alpha" %}
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/health-problems.html]: Double dash

File: views/conditions/type-2-diabetes/understanding-medication.html [hunk 12]
```diff
@@ -0,0 +1,154 @@
+{% extends "_layouts/_layout-default.html" %}
+
+{% set pagetitle = "Understanding medication - Type 2 diabetes - - NHS.UK Alpha" %}
```
Comment (Demotive) [inline views/conditions/type-2-diabetes/understanding-medication.html]: Double dash

File: views/conditions/type-2-diabetes/finding-help-and-support.html (hunk 7 above)
Comment (Demotive) [inline views/conditions/type-2-diabetes/finding-help-and-support.html:50]: Not to worry - it's not going to do any harm in /design

**Pull request merged**